src/current/v23.1/changefeed-sinks.md

```diff
@@ -261,9 +261,10 @@ For a list of compatible parameters and options, refer to [Parameters]({% link {
 
 ### Pub/Sub sink configuration
 
-The `pubsub_sink_config` option allows the changefeed flushing and retry behavior of your Pub/Sub sink to be configured.
+You can configure flushing, retry, and concurrency behavior of changefeeds running to a Pub/Sink sink with the following:
```

Comment:
I assume 'Pub/Sink' is an intentional change from 'Pub/Sub sink' ?

Reply:
No, definitely not, this is a typo!!!!